core/src/main/resources/hudson/util/HudsonIsLoading/index.jelly

```diff
@@ -25,12 +25,14 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:x="jelly:xml" xmlns:f="/lib/form">
-
+    <j:new var="h" className="hudson.Functions"/><!-- instead of JSP functions -->
```

Comment:
only NIT here is everything is jelly, so no need to have `<!-- instead of JSP functions -->`

Reply:
Good call, I just copy and pasted from somewhere else and didn't think about that

Reply:
I figured as much, was just an observation

Reply:
```$ ag 'className="hudson.Functions"' | cut -d: -f 3- | sed -e 's/^[[:space:]]*//' | sort | uniq -c                                                                  
      4 <j:new var="h" className="hudson.Functions" />
      7 <j:new var="h" className="hudson.Functions" /><!-- instead of JSP functions -->
      3 <j:new var="h" className="hudson.Functions"/><!-- instead of JSP functions -->
      1 <j:new var="h" className="hudson.Functions" /><!-- needed for printing title. -->
```

hah, apparently its super common